boards/winterfell/winterfell.config

```diff
@@ -52,7 +52,7 @@ dxe_offset := 860000
 dxe_size := 6a0000
 flash-dxe: $(build)/$(BOARD)/linuxboot.rom
 	( echo u$(dxe_offset) $(dxe_size) ; \
-	pv $(build)/linuxboot-git/build/$(BOARD)/dxe.vol \
+	pv $(build)/linuxboot-b5376a441e8e85cbf722e943bb8294958e87c784/build/$(BOARD)/dxe.vol \
```

Comment:
Ok, I get what you are doing here.
You can get the build directory by reusing Makefile included module.

In this case: `$(linuxboot_base_dir)`